src/SarifTests/SarifUtilitiesTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.CodeAnalysis.Sarif;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Sarif.Tests
+{
+    [TestClass()]
+    public class SarifUtilitiesTests
+    {
+        private const string V1_0_0 = "1.0.0";
+        private const string V1_0_0_BETA_5 = "1.0.0-beta.5";
+
+        [TestMethod()]
+        public void ConvertToSchemaUriTest1()
+        {
+            V1_0_0.ConvertToSarifVersion().ConvertToSchemaUri();
+        }
+
+        [TestMethod()]
+        public void ConvertToSchemaUriTest2()
+        {
+            V1_0_0_BETA_5.ConvertToSarifVersion().ConvertToSchemaUri();
+        }
```

Comment:
Now of course, as the product code stands today, this test will fail (once it's rewritten as above). That means you can't check these tests in as is.

You have two choices:

* Mark this test as Ignored, check in, and then immediately submit a separate PR for the product code change.

```
[TestMethod, Ignore]
```

* (Preferable) Make the product code change in the same PR where you introduce the failing test -- that will make the test pass.